### PR TITLE
Improve overview article for SanAPI

### PR DESCRIPTION
**Pull request by haritonasty**
Add a link to sanpy
  Add "let's start" section

### CODE REVIEW

File: src/docs/for-developers/index.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+Also we have a python wrapper for our graphql API, usage of this library is even
+easier.
+
+[Sanpy library](https://github.com/santiment/sanpy)
+
+```
+pip install sanpy
+```
+
+And you can try to get the same ethereum team dev activity data with this python library
```
Comment (IvanIvanoff) [inline src/docs/for-developers/index.md]: If it's the same as the one above why the results are so different? one of the results is wrongly copy-pasted form something else

File: src/docs/for-developers/index.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+)
+
+print(report)
+```
+
+Example result:
+
+```
+                           activity
+datetime
```
Comment (IvanIvanoff) [inline src/docs/for-developers/index.md]: both datetime and activity should be on the same row

File: src/docs/for-developers/index.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+2019-01-03 00:00:00+00:00       140
+2019-01-04 00:00:00+00:00       177
+2019-01-05 00:00:00+00:00        46
+2019-01-06 00:00:00+00:00        22
+2019-01-07 00:00:00+00:00       552
+```
+
+## Authentication
+
+Some of the APIs require to have a valid API key, belonging to an account with a
```
Comment (IvanIvanoff) [inline src/docs/for-developers/index.md]:
```suggestion
Some of the APIs require a valid API key, belonging to an account with a
```

File: src/docs/for-developers/index.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+paid subscription to access all the data. The API key can be generated on your
```
Comment (IvanIvanoff) [inline src/docs/for-developers/index.md]:
```suggestion
paid subscription to access more data. The API key can be generated on your
```
some paid plans don't have access to all data, just more than free tier

File: src/docs/for-developers/index.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+      "message": "In argument \"to\": Expected type \"DateTime!\", found null."
+    }
+  ]
+}
+```
+
+If the query does not return status code `200`, then something else is happening. Here are some of the options:
+
+- `429` - you are being rate limited. Reduce the amount of requests you are doing
+- `502` - an internal server error has occured. Let us know at [https://santiment.net/discord](https://santiment.net/discord)
```
Comment (IvanIvanoff) [inline src/docs/for-developers/index.md]:
```suggestion
- `5xx` - an internal server error has occured. Let us know at [https://santiment.net/discord](https://santiment.net/discord)
```

File: src/docs/for-developers/index.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+  or at the `#support channel`
+
+## Live API Explorer
+
+There is a live explorer, where you can run queries directly from the browser. The
+explorer is accessible here: [https://api.santiment.net/graphiql](https://api.santiment.net/graphiql)
+
+Here is an example of running a query and seeing the results directly in the browser:
+
+[https://api.santiment.net/graphiql?query=%7B%0A%20%20transactionVolume(slug%3A%22santiment%22%2C%20from%3A%222019-01-01T00%3A00%3A00Z%22%2C%20to%3A%222019-03-01T00%3A00%3A00Z%22)%20%7B%0A%20%20%20%20datetime%0A%20%20%20%20transactionVolume%0A%20%20%7D%0A%7D](<https://api.santiment.net/graphiql?query=%7B%0A%20%20transactionVolume(slug%3A%22santiment%22%2C%20from%3A%222019-01-01T00%3A00%3A00Z%22%2C%20to%3A%222019-03-01T00%3A00%3A00Z%22)%20%7B%0A%20%20%20%20datetime%0A%20%20%20%20transactionVolume%0A%20%20%7D%0A%7D>)
```
Comment (IvanIvanoff) [inline src/docs/for-developers/index.md]: For better UX we can have [GraphQL Request fetching transaction volume](<actual url goes here>) - the same can be applied to all links

File: src/docs/sanapi/index.md [hunk 7]
```diff
@@ -5,20 +5,72 @@ date: "2019-09-20"
...
+)
+
+print(report)
+```
+
+Example result:
+
+```
+                           activity
+datetime
```
Comment (IvanIvanoff) [inline src/docs/sanapi/index.md]: datetime and activity should be on the same row

File: src/docs/for-developers/index.md (hunk 2 above)
Comment (Partysun) [reply src/docs/for-developers/index.md]: It's the same as real answer. It's not on the same.

**Pull request merged**